.github/workflows/pypackaging.yml

```diff
@@ -0,0 +1,90 @@
+name: Python Packaging
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
We want release_XXYY branches here too

Reply:
Releases were supposed to be covered by the case below, when you manually create a release in the UI.  If that's not how releases get made, then yes, we can add some push rules here.  I would think it would be on release tags rather than branches though.

Reply:
I Was thinking more on always running these workflow and when a env var is passed to the workflow it uploads it to pypi

Reply:
Ah, ok.  I think we can do that.